earth2mip/inference_ensemble.py

```diff
@@ -264,7 +264,19 @@ def perturb(x, rank, batch_id, device):
             [channel_stds[channel] for channel in model.in_channel_names],
             device=x.device,
         )
-        x += noise * scale[:, None, None]
+
+        if config.ic_perturbed_channels == "all":
+            x += noise * scale[:, None, None]
+        else:
+            if not isinstance(config.ic_perturbed_channels, list):
+                config.ic_perturbed_channels = [config.ic_perturbed_channels]
```

Comment:
avoid mutating the config object. The user won't expect run_ensemble to change the config they pass.